src/Pages/Templates/TemplateDetails/components/Tables/TemplateRepositoriesTable.tsx

```diff
@@ -74,7 +74,7 @@ export default function TemplateRepositoriesTable({
   }, [snapshotList.length]);
 
   const columnHeaders = [
-    { name: 'Repository Name' },
+    { name: 'Repository' },
```

Comment:
That should be "Name" I think. Check the Jira again.

Reply:
Lol i think i got lost in code 

Right this needs to be "Name"!!